docker/genomes-catalog-update/scripts/annot_gff.py

```diff
@@ -46,6 +46,51 @@ def get_eggnog(eggnot_annot):
     return eggnogs
 
 
+def get_sanntis(sanntis_file, prokka_gff):
+    cluster_positions = dict()
+    sanntis_result = dict()
+    bgc_annotations = dict()
+    # save positions of each BGC cluster annotated by SanntiS to dictionary cluster_positions
+    # and save the annotations to dictionary sanntis_result
+    with open(sanntis_file, "r") as sanntis_in:
+        for line in sanntis_in:
+            if not line.startswith("#"):
+                cols = line.strip().split("\t")
```

Comment:
A more readable way of splitting this:
```python
contig, _, _ , start_pos, end_pos, _, _, annotations = line.strip().split("\t")
```
that way you can "spread" the values, and assign them directly to variables (the "_" are just unused variables)

Reply:
great suggestion, thank you! added the change